src/python/WMCore/WMSpec/StdSpecs/StdBase.py

```diff
@@ -367,10 +367,16 @@ def setupProcessingTask(self, procTask, taskType, inputDataset=None, inputStep=N
         procTaskCmsswHelper = procTaskCmssw.getTypeHelper()
         procTaskStageHelper = procTaskStageOut.getTypeHelper()
 
-        if 'Multicore' in taskConf and taskConf['Multicore'] > 0:  # used for StepChain overrides
-            procTaskCmsswHelper.setNumberOfCores(taskConf['Multicore'])
-        else:
-            procTaskCmsswHelper.setNumberOfCores(self.multicore)
+        # StepChain overrides
+        multicore = self.multicore
+        eventStreams = self.eventStreams
+        if 'Multicore' in taskConf and taskConf['Multicore'] > 0:
+            multicore = taskConf['Multicore']
+        if 'EventStreams' in taskConf and taskConf['EventStreams'] >= 0:
+            eventStreams = taskConf['EventStreams']
```

Comment:
If taskConf can over write that, (i.e, different task can have different EventStreams value,

Shouldn't following lines modify to add that?
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/WMSpec/StdSpecs/StepChain.py#L209
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/WMSpec/StdSpecs/StepChain.py#L435

How about the TaskChain? Doesn't it have to use this value as well?
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/WMSpec/StdSpecs/TaskChain.py#L644